pkgs/stdenv/generic/make-derivation.nix

```diff
@@ -381,7 +381,16 @@ else let
             "-DCMAKE_HOST_SYSTEM_VERSION=${stdenv.buildPlatform.uname.release}"
           ];
         in
-          explicitFlags ++ lib.optionals (stdenv.hostPlatform != stdenv.buildPlatform) crossFlags;
+          explicitFlags
+          ++ lib.optionals (stdenv.hostPlatform != stdenv.buildPlatform) crossFlags
+          ++ lib.optionals (stdenv.isEmscripten && stdenv ? cc.cc) [
+
+             # TODO ATN: I'm not sure where to put this, it doesn't look like it belongs here
+            "-DCMAKE_TOOLCHAIN_FILE=${stdenv.cc.cc}/share/emscripten/cmake/Modules/Platform/Emscripten.cmake"
```

Comment:
Does a "native" Emscripten build make sense (running Nix on Emscripten)?  If not, put this in crossFlags above, where it'll be at home next to the Darwin one.